images/php/cli/55-cli-helpers.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+dsql () {
+	drush sql-sync $1 default -d -v
```

Comment:
let's do them without `-d -v` I think that's too verbose, if somebody needs the verbose, they can still run `dsql -d -v`